src/nfa/shufti_simd.hpp

```diff
@@ -233,15 +234,15 @@ const u8 *fwdBlockDouble(SuperVector<S> mask1_lo, SuperVector<S> mask1_hi, Super
     SuperVector<S> t = t1 | (t2 >> 1);
 
     typename SuperVector<S>::movemask_type z = t.eqmask(SuperVector<S>::Ones());
-    DEBUG_PRINTF(" z: 0x%016llx\n", (u64a)z);
+    DEBUG_PRINTF("    z: 0x%08x\n", z);
```

Comment:
unfortunately the casting has to stay, otherwise DEBUG_PRINTF() fails on AVX512 which uses u64a as movemask_type.

Reply:
An accident from rebasing sorry.